src/main/java/com/github/dockerjava/core/command/CreateContainerCmdImpl.java

```diff
@@ -590,6 +596,13 @@ public CreateContainerCmd withCpuPeriod(Integer cpuPeriod) {
     }
 
     @Override
+    public CreateContainerCmd withCpuQuota(Integer cpuQuota) {
+        checkNotNull(cpuQuota, "cpuQuota was not specified");
+        hostConfig.withCpuQuota(cpuQuota);
```

Comment:
old methods should be removed in future, now still exists to not break compatibility. Adding newer makes no sense if it possible to directly use hostconfig.